client/src/components/reviews/breakdownComponents/BarGraph.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {BarForGraph} from '../sharedStyles/sharedStyledComponents'
+
+export const BarGraph = ({percent}) => {
+  let starPercent = percent.toFixed(0);
+  let styling = {
+    width: `${starPercent}%`
+  }
+  return (<BarForGraph style={styling} />);
+}
```

Comment:
Is that all that is required to create a graph? That is amazing.

Reply:
Yes, it is simply two styled divs. One that sets the original width, and a second one that is nested inside (width set to the percent of the reviews)